src/udev/udevd.c

```diff
@@ -768,10 +768,7 @@ static int on_kill_workers_event(sd_event_source *s, uint64_t usec, void *userda
 static int manager_disable_kill_workers_event(Manager *manager) {
         int r;
 
-        if (!manager->kill_workers_event)
-                return 0;
-
-        r = sd_event_source_set_enabled(manager->kill_workers_event, SD_EVENT_OFF);
+        r = event_source_disable(manager->kill_workers_event);
         if (r < 0)
                 return log_warning_errno(r, "Failed to disable event source for cleaning up idle workers, ignoring: %m");
 
```

Comment:
Maybe just kill this function completely now? It it just a wrapper for a wrapper and doesn't do anything.